integration-tests/dist/index.js

```diff
@@ -57405,7 +57405,7 @@ function patch (fs) {
   // created files.  Try again on failure, for up to 60 seconds.
 
   // Set the timeout this long because some Windows Anti-Virus, such as Parity
-  // bit9, may lock files for up to a minute, causing npm package install
+  // bit9, may lock files for up to a minute, causing _getSpawnOptions(options, toolPath)npm package install
```

Comment:
? do you know where this is coming from ?

This string is part of the instructions to fix an issue with child.spawn https://github.com/Jahia/jahia-modules-action/tree/main/integration-tests#fix-issue-with-childspawn, could it be that it was added in this location by mistake